testing/constants.ts

```diff
@@ -1,3 +1,12 @@
 export enum HttpStatusCode {
   Unauthorized = 401,
 }
+
+export const DEFAULT_ALLOWED_GET_DOMAINS = [
```

Comment:
can you leave a note here to keep it in sync with the runtime config we use to actually gate it?